### ISSUE TITLE
delegateEvents is missing on server causing a rollup build error

### ISSUE THREAD

**Issue by btakita** [solidjs/solid#1200]
### Describe the bug

When building the project on vercel, an error occurs.

```
Error: 'delegateEvents' is not exported by ../../node_modules/.pnpm/solid-js@1.5.3/node_modules/solid-js/web/dist/server.js, imported by .solid/server/entry-server.js
```

This is coming from solid-router, which calls delegateEvent in the client logic. This causes `npm run build` to fail since the empty delegateEvent function does not exist.

### Your Example Website or App

https://github.com/btakita/wholly-trinity/

### Steps to Reproduce the Bug or Issue

Go to a solid-start project using solid-router.
pnpm run build

### Expected behavior

The build should be successful.

### Platform

- OS: Linux
- Browser: N/A
- Version: 1.5.3

**Comment by ryansolid** [comment]
fixed 1.5.4

**Comment by ryansolid** [comment]
Yeah we can stub it. I would have hoped it'd get tree shaken but we can always just add an no op function.
